src/Presentation/Nop.Web/Areas/Admin/Views/Shared/Components/MultistorePreview/Default.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model IList<MultistorePreviewModel>
+
+<div class="btn-group" id="multiStorePreview">
+    <button type="button" class="btn btn-info">
+        <i class="far fa-eye"></i>
+        @T("Admin.Common.Preview")
+    </button>
+    <button type="button" class="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+        <span class="caret"></span>
+        <span class="sr-only">&nbsp;</span>
+    </button>
```

Comment:
I was wondering if Entities that are limited to a single store (or there is only a single store) should have a standard button (Or make this section of the button clickable for current store) as opposed to having to do 2 clicks now?